README.md

```diff
@@ -22,4 +22,19 @@ mkctr \
 `mkctr` auto discovers `GOOS`/`GOARCH` from the specified base image. If the base image supports multiple platforms, binaries are compiled for each platform as long as it's one of `linux/amd64`, `linux/386`, `linux/arm`, `linux/arm64`.
 
 ## Maturity
-This is under active development. While Tailscale uses it, backwards compatability is not guaranteed, and some functionality is missing.
\ No newline at end of file
+
+This is under active development. While Tailscale uses it, backwards compatability is not guaranteed, and some functionality is missing.
+
+## Known issues
```

Comment:
I ran into this whilst trying to build a custom base for containerboot image and thought it'd be useful to add. Really, we should fix it, but might be low priority given that this is largely just for our images. Happy to not add this commit though if people don't like it

Reply:
Maybe file a tracking issue instead?

Reply:
https://github.com/tailscale/mkctr/issues/13
seems like it is to do with the manifest structure, not with the type